src/mediaControllers/chat.js

```diff
@@ -23,37 +23,39 @@
     var logComponent = connect.LogComponent.CHAT;
 
     var createMediaInstance = function () {
-      publishTelemetryEvent("Chat media controller init", mediaInfo.contactId);
```

Comment:
looks like linting applied to these files, but are these synced with the internal changes?